src/variables.jl

```diff
@@ -27,12 +34,18 @@ function JuMP.build_variable(
         _error("Invalid fix value, must be false or true.")
     elseif info.has_start && !isone(info.start) && !iszero(info.start)
         _error("Invalid start value, must be false or true.")
+    elseif (info.has_start || info.has_fix) && !isnothing(logical_compliment)
```

Comment:
Why? Is it because we haven't added this support yet or because it cannot be done?

Reply:
Because a logical compliment variable Y2 actually maps to 1 - Y1. Any start value or fix value with Y2 would conflict any defined in Y1. We could add a bunch of checks to avoid such conflicts, but that seems unnecessary. I clarified the error message to make this more clear.